docs/specviz/displaying.rst

```diff
@@ -29,7 +29,68 @@ Data can be selected and de-selected by clicking on the "hammer and screwdriver"
 Pan/Zoom
 ========
 
-Coming soon
+Included with the Specviz tool are multiple manners to change your view/perspective on the data displayed, depending on your environment:
+
+Interactive Pan/Zoom (Desktop App interface)
+--------------------------------------------
+You can find the following Pan/Zoom tools available in the tool drawer of the viewer:
+
+2D Bidirectional Pan/Zoom
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The 2D Pan/Zoom Icon allows you to zoom using the scroll wheel. The window will zoom into the area around your cursor:
+
+.. image:: ../img/SpecViz_2D_Zoom.gif
+    :alt: Zooming in Specviz with the Scroll Wheel
+
+To pan, simply click and drag the window
+
+Horizontal/Vertical Zoom
+^^^^^^^^^^^^^^^^^^^^^^^^
+The Horizontal and Vertical Zoom tools allow you to zoom along each axis, while locking the other. Zoom by scrolling
+
+.. note::
+    A gif here showing how these tools manipulate the view, similar to above may be useful here
+
+
+API Pan/Zoom (Notebook interface)
+---------------------------------
+The Specviz helper contains a set of convinence methods to programmatically define the view of the spectrum viewer.
+
+For information about the Specviz, and other, helpers, and how to instantiate one, please see `helpers <https://readthedocs.org/404>`_
+
+.. warning::
+    The helper page has not yet been written/defined. Until that documentation is written, you may instantiate a Specviz Helper via:
+    ::
+
+>>> from jdaviz import SpecViz
+>>> # Instantiate an instance of SpecViz
+>>> SpecViz = SpecViz()
+>>> # Display SpecViz
+>>> SpecViz.app
+
+Limit methods
+^^^^^^^^^^^^^
+You can use the methods x_limits() and y_limits() to modify the field of view of Specviz. You can provide it a scalar (which assumes the units of the loaded spectra), an Astropy Quantity, or 'auto' to automatically scale
+::
+
+>>> SpecViz.x_limits()
+>>> SpecViz.x_limits(650*u.nm,750*u.nm)
+>>> SpecViz.y_limits('auto', 110.0)
+
+Additionally, you can provide the limit methods with a `~specutils.SpectralRegion`. Specviz shall set the bounds the upper and lower bounds of the SpectralRegion
+
+>>> from astropy import units as u
+>>> from specutils.spectral import SpectralRegion
+>>> bounds = SpectralRegion(0.45*u.nm, 0.6*u.nm)
+>>> SpecViz.x_limits(bounds)
+
+Autoscale methods
+^^^^^^^^^^^^^^^^^
+You can also quickly return to the default zoom using :py:func:`SpecViz.autoscale_x()` and :py:func:`SpecViz.autoscale_y()`
```

Comment:
I think we decided to go with `Specviz` in the documentation instead of `SpecViz`.

Reply:
Agreed, but as I brought up with @eteq, our helper class is still called `SpecViz` so wherever I refer to the helper object, I continue to use `SpecViz`. Unless you think this isn't appropriate here?

Reply:
Use SpecViz as the name of the helper in docs until it gets changed in the code.